example/navigators/MainNavigator.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { createStackNavigator } from '@react-navigation/stack';
+import ExamplesListScreen from '../screens/ExamplesListScreen';
+import { CONCRETE_EXAMPLE_SCREENS } from '../constants/ConcreteExampleScreens';
+import { Providers } from 'expo-maps/build/Maps.types';
+
+// TODO: definetly type this better!
+interface ProviderProp {
+  provider: Providers;
+}
+
+export type ExamplesStackNavigatorProps = {
+  ExamplesListScreen: undefined;
+  Markers: undefined;
+  Polygons: undefined;
+  Polylines: undefined;
+  Controls: undefined;
+  'Google Maps Styling': undefined;
+  Gestures: undefined;
+  'Map Types': undefined;
+};
+
+const ExamplesStackNavigator =
+  createStackNavigator<ExamplesStackNavigatorProps>();
+
+export default function MainNavigator() {
+  return (
+    <ExamplesStackNavigator.Navigator>
+      <ExamplesStackNavigator.Screen
+        name="ExamplesListScreen"
+        component={ExamplesListScreen}
+        options={{ title: 'ExpoMaps 🗺️' }}
+      />
+      {CONCRETE_EXAMPLE_SCREENS.map(({ name, screen }) => (
+        <ExamplesStackNavigator.Screen
+          name={name}
+          component={screen}
+          key={name}
+        />
+      ))}
```

Comment:
I would suggest adding:
`headerStyle: {   borderBottomWidth: 1,   }`
to every ExamplesStackNavigator.Screen option prop

also, back button icon on the appbar is not visible on android